### PR TITLE
Feat/update libs

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- going and updating deps across whole monorepo
- [Notion](https://www.notion.so/satoshilabs/Dependency-Management-1b5bf845aa1f4ca7b9d57ea9ccd3fe63?p=069bc48b07404e4696fbaf66712f4f15&pm=s)

## Related Issue
Sprintly check of dependencies

## Questions to discuss
- what to do with ESM deps?

### CODE REVIEW

**Comment by tomasklim** [comment]
> ### [Socket Security Pull Request Report](https://socket.dev)
> Dependency issues detected. If you merge this pull request, you will not be alerted to the instances of these issues again.
> 
> ** 😵‍💫 Bin script confusion **
> Package	Bin script	Source
> [@ethereumjs/rlp@4.0.0](https://socket.dev/npm/package/@ethereumjs/rlp/overview/4.0.0) (added)	[`rlp`](https://socket.dev/npm/package/@ethereumjs/rlp/files/4.0.0/bin/rlp)	[`packages/connect-plugin-ethereum/package.json`](https://github.com/trezor/trezor-suite/pull/6992/files#diff-82c9923ad03a0aa12f63ff124a9d81b15aca41bcc417bc1bd027b9bcd20fdaed) via [@metamask/eth-sig-util@5.0.2](https://socket.dev/npm/package/@metamask/eth-sig-util/overview/5.0.2), [@ethereumjs/util@8.0.2](https://socket.dev/npm/package/@ethereumjs/util/overview/8.0.2),[`packages/suite/package.json`](https://github.com/trezor/trezor-suite/pull/6992/files#diff-cf2f50ab25c7c54b323f082bf9c7c129fbc05b5d6e7e43371817b57149847bbe) via [@ethereumjs/tx@4.0.1](https://socket.dev/npm/package/@ethereumjs/tx/overview/4.0.1)
> [rlp@2.2.7](https://socket.dev/npm/package/rlp/overview/2.2.7) (added)	[`rlp`](https://socket.dev/npm/package/rlp/files/2.2.7/bin/rlp)	[`packages/suite/package.json`](https://github.com/trezor/trezor-suite/pull/6992/files#diff-cf2f50ab25c7c54b323f082bf9c7c129fbc05b5d6e7e43371817b57149847bbe) via [ethereumjs-util@7.1.5](https://socket.dev/npm/package/ethereumjs-util/overview/7.1.5),[`suite-common/wallet-utils/package.json`](https://github.com/trezor/trezor-suite/pull/6992/files#diff-11581fd60f0294d0c0c740a841f63ae44ac7ca5d69935ab71fbdc02ad2d3d1cb) via [ethereumjs-util@7.1.5](https://socket.dev/npm/package/ethereumjs-util/overview/7.1.5)
> **Pull request report summary**
> **Bot Commands**
> _Powered by [socket.dev](https://socket.dev)_

Looks like a false alarm or I am not sure where is the duplication of bin scripts

**Comment by Nodonisko** [comment]
@tomasklim Thanks for verification. As I said, I would not call it alarm, it's just insight that you should verify in person.

File: yarn.lock [hunk 1]
```diff
@@ -447,6 +572,24 @@ __metadata:
...
+"@babel/parser@npm:^7.20.1":
+  version: 7.20.1
+  resolution: "@babel/parser@npm:7.20.1"
+  bin:
+    parser: ./bin/babel-parser.js
+  checksum: 2db7ba692b8b3054df129876b115ed8b265d5c138dee5e1db56a999209df3dd3d508e0985bf3cc44fc432498da094c164906b531d049608e55e208dae0678d42
+  languageName: node
+  linkType: hard
+
+"@babel/parser@npm:^7.20.2":
```
Comment (Nodonisko) [inline yarn.lock]: couldn't we consolidate these two version? (the one above seems to be different only by path number)

File: yarn.lock [hunk 2]
```diff
@@ -347,6 +452,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/helper-simple-access@npm:^7.20.2":
```
Comment (Nodonisko) [inline yarn.lock]: couldn't we consolidate with one above? it's seems to different only in minor version

File: yarn.lock [hunk 3]
```diff
@@ -288,6 +354,38 @@ __metadata:
...
+    "@babel/helper-split-export-declaration": ^7.18.6
+    "@babel/helper-validator-identifier": ^7.19.1
+    "@babel/template": ^7.18.10
+    "@babel/traverse": ^7.19.6
+    "@babel/types": ^7.19.4
+  checksum: c28692b37d4b5abacc775bcab52a74f44a493f38c58cb72b56a6c6d67a97485dd8aff6f26905abd1a924d3261a171d0214a9fb76f48d8598f1e35b8b29284792
+  languageName: node
+  linkType: hard
+
+"@babel/helper-module-transforms@npm:^7.20.2":
```
Comment (Nodonisko) [inline yarn.lock]: couldn't we consolidate?

File: yarn.lock [hunk 4]
```diff
@@ -123,6 +153,28 @@ __metadata:
...
+  resolution: "@babel/generator@npm:7.20.1"
+  dependencies:
+    "@babel/types": ^7.20.0
+    "@jridgewell/gen-mapping": ^0.3.2
+    jsesc: ^2.5.1
+  checksum: e6846d88c59a5dae4c86f3cc84f84972cf9cc5fa0f4944606303a9df3ba1be388e0cb08a625c86f7282ab03faf54acd72efba34f019b6762f4739a175173783e
+  languageName: node
+  linkType: hard
+
+"@babel/generator@npm:^7.20.2":
```
Comment (Nodonisko) [inline yarn.lock]: couldn't we consolidate? it seems to differ only by path version

**Review by Nodonisko — CHANGES_REQUESTED** [review]
Good work, my only concern is that now there is lot of version that differs only by patch number or something can you please verify why they did not deduplicate? It think yarn 2 should do this automatically but maybe somewhere are exact version specified

PS I added comments to only few of them but there is more.

File: yarn.lock (hunk 2 above)
Comment (tomasklim) [reply yarn.lock]: deduplicated by `yarn dedupe` I thought that we had a dedupe checker in CI
14130e270e0503b7928ec87b329977535de60eda

**Comment by tomasklim** [comment]
> Good work, my only concern is that now there is lot of version that differs only by patch number or something can you please verify why they did not deduplicate? It think yarn 2 should do this automatically but maybe somewhere are exact version specified
> 
> PS I added comments to only few of them but there is more.

https://yarnpkg.com/cli/dedupe#in-depth-explanation

**Comment by Nodonisko** [comment]
Thanks I did not know that! 

We should integrate this dedupe check to CI as it was before. Can you do that? (of course it can be follow-up)

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**